backend/infrahub/core/relationship/model.py

```diff
@@ -218,7 +231,7 @@ async def get_node(self, db: InfrahubDatabase) -> Node:
         node: Node = await registry.manager.get_one_by_id_or_default_filter(
             db=db,
             id=self.node_id,
-            kind=self.schema.kind,
+            kind=str(self.schema.kind.value),
```

Comment:
Is the `str()` required here? Also did this cause any problems anywhere or did we perhaps convert the enum to a string later? Was it just mypy that started to complain?